db/db.js

```diff
@@ -424,9 +437,9 @@ async function startChatbotSessionState(session) {
 }
 
 // Retrieves the data from the locations table for a given location
-async function getLocationData(location) {
+async function getLocationData(locationid) {
     try{
-        const results = await pool.query('SELECT * FROM locations WHERE locationid = $1', [location]);
+        const results = await pool.query('SELECT * FROM locations WHERE locationid = $1', [locationid]);
         if(results == undefined){
             return null;
         }
```

Comment:
Should this one also use `createLoctionFromRow`?

Reply:
I only changed the things that needed to be updated to accommodate Location as an object vs the returned rows

Reply:
Fair enough :+1: